tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -66,11 +64,11 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
 
     "successfully create a 2nd new user" in {
       val jwt = JwtHelper.generateValidJwt(secondaryTestUserId.getId)
-      val resp = Try(IronSdk.userCreate(jwt, secondaryTestUserPassword, new UserCreateOpts())).toEither
+      val resp = Try(IronSdk.userCreate(jwt, secondaryTestUserPassword, UserCreateOpts.create(true))).toEither
```

Comment:
I know this isn't necessarily directly related to this PR, but is there a reason `UserCreateOpts.create(..)` is used rather than `UserCreateOpts(...)`. Are we not able to define multiple Java constructors?

Reply:
I tried making it a second constructor and it seems to work. Should I make that change as part of this PR? That would make this a breaking change, right?

Reply:
If it's only UserCreateOps.create and not others, I would just change it here. If there are others it would be good to write a separate ticket to get that cleaned up.

Your call.

Reply:
It's on all the `*CreateOpts`, so I'm not going to include it as part of this.